test/tstGeometryBarycentricCoordinates.cpp

```diff
@@ -22,53 +22,55 @@
 
 namespace tt = boost::test_tools;
 
-template <int N>
-using Array = Kokkos::Array<float, N>;
+using CoordinatesList = std::tuple<float, double>;
 
-BOOST_AUTO_TEST_CASE(barycentric_triangle)
+BOOST_AUTO_TEST_CASE_TEMPLATE(barycentric_triangle, Coordinate, CoordinatesList)
 {
   using ArborX::Point;
   using ArborX::Triangle;
   using ArborX::Experimental::barycentricCoordinates;
+  using Array = Kokkos::Array<Coordinate, 3>;
```

Comment:
Did you mean to vary the element type in the array but keep the same geometry coordinates type?